udata_front/theme/gouvfr/assets/js/components/search/OrganizationSearch/OrganizationSearch.vue

```diff
@@ -91,4 +93,15 @@ watchDebounced(q, async (newValue, oldValue) => {
   .shadow {
     box-shadow: 0px 4px 2px var(--border-default-grey);
```

Comment:
```suggestion
    box-shadow: 0px 4px 4px var(--border-default-grey);
```

This class should be move to a LESS file to be used in other components too